src/components/Booklist.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+import Book from './Book';
+
+const Booklist = ({ books }) => (
+  <div className="book-list">
+    <ul>
+      {books.map((book) => (
+        <Book key={book.id} title={book.title} author={book.author} />
+        // <p>Book</p>
```

Comment:
- [ ] You should remove unused code from your code because it will make code base cleaner plus it is not the best practice to leave unused code in the code base.